Changes

```diff
@@ -37,6 +37,10 @@ Next version (4.05.0):
 
 ### Compiler user-interface and warnings:
 
+- PR#7460, GPR#1011: catch exception thrown when unknown files are passed
+  as argument.
+  (Bernhard Schommer)
```

Comment:
Maybe point out that @Octachron was helpful in reviewing? (Or maybe you had found the nasty syntax thing independently?) Otherwise it now seems good to merge.